test/test_qml_editorwidgets.cpp

```diff
@@ -35,12 +33,7 @@ class Setup : public QObject
   public slots:
     void qmlEngineAvailable( QQmlEngine *engine )
     {
-      Q_UNUSED( engine )
-
-      REGISTER_SINGLETON( "org.qfield", LayerUtils, "LayerUtils" );
-
-      qRegisterMetaType<QgsField>( "QgsField" );
-      qRegisterMetaType<QVariant::Type>( "QVariant::Type" );
+      qmlInit( engine );
```

Comment:
Nice.
Would it make sense to share that with app?

Reply:
@m-kuhn , I was thinking about this. I was 50/50, unsure. I'm happy to implement it if you feel it's the right thing to do.

Reply:
Let's keep it out for now, that would ideally be done as plugin init code.